api/controllers/comment.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+const winston = require('winston');
+const request = require('request');
+const axios = require('axios');
+
+const config = require('../config');
+const logger = winston.createLogger(config.logger.winston);
+const db = require('../models');
+const common = require('../common');
+
+
+/**
+ * @apiGroup Comment
+ * @api {get} /comment  Get Comment List of Particular Project
+ * @apiHeader {String} authorization 
+ *                              A valid JWT token "Bearer: xxxxx"
+ * @apiSuccess {Object}            List of Comments 
+ */
+
+router.get('/', common.jwt({credentialsRequired: true}), (req, res, next) =>{
+    if(!req.query.project) return next("project not defined");
+    let project = req.query.project;
+    if(project) {
+        const find = {"project": project, "removed" : false};
+        db.Comments.find(find).exec((err, recs)=>{
+            // console.log(recs);
+            if(err) return next(err);
+            // console.log(recs);
+            db.Comments.countDocuments(find).exec((err, count)=>{
+                if(err) return next(err);
+                res.json({
+                    comments : recs, 
+                    count
+                });
+            })
+        })
+    }
+});
+
+/**
+ * @apiGroup Comments
+ * @api {post} /comment Post comment on a Project
+ * @apiParam {user_id} userID 
+ * @apiParam {String} comment
+ * @apiParam {project} projectID
+ * @apiSuccess {}            Comment registered 
+ */
+
+router.post('/', common.jwt(), function(req, res, next) {
+    if(!req.body.user_id) return next('user id missing');
+    if(!req.body.comment) next("comment string missing");
+    if(!req.body.project) next("project id missing");
+    let comment = new db.Comments(req.body);
```

Comment:
Similar to get, we should create separate API for each entities, I think. 

Apply access control. I think having read access to the project should allow them to add comments (so non-member can comment on it)